shared/lib/error-utils.js

```diff
@@ -3,17 +3,27 @@ import getFirstPreferredLangCode from '../../app/scripts/lib/get-first-preferred
 import { fetchLocale, loadRelativeTimeFormatLocaleData } from '../modules/i18n';
 import switchDirection from './switch-direction';
 
+const defaultLocale = 'en';
 const _setupLocale = async (currentLocale) => {
-  const currentLocaleMessages = currentLocale
-    ? await fetchLocale(currentLocale)
-    : {};
-  const enLocaleMessages = await fetchLocale('en');
+  const enRelativeTime = loadRelativeTimeFormatLocaleData(defaultLocale);
+  const enLocale = fetchLocale(defaultLocale);
 
-  await loadRelativeTimeFormatLocaleData('en');
-  if (currentLocale) {
-    await loadRelativeTimeFormatLocaleData(currentLocale);
+  const promises = [enRelativeTime, enLocale];
+  if (currentLocale === defaultLocale) {
+    // enLocaleMessages and currentLocaleMessages are the same; reuse enLocale
+    promises.push(enLocale); // currentLocaleMessages
+  } else if (currentLocale) {
+    // currentLocale does not match enLocaleMessages
+    promises.push(fetchLocale(currentLocale)); // currentLocaleMessages
+    promises.push(loadRelativeTimeFormatLocaleData(currentLocale));
+  } else {
+    // currentLocale is not set
+    promises.push(Promise.resolve({})); // currentLocaleMessages
   }
 
+  const [, enLocaleMessages, currentLocaleMessages] = await Promise.all(
+    promises,
+  );
```

Comment:
This is really two changes:

1. don't fetch the `"en"` locale twice
2. if we have to fetch multiple locales, do them at the same time.